packages/apis/DEVELOPMENT.md

```diff
@@ -10,12 +10,15 @@ $ yarn build
 
 ## Re-generate APIs code
 
+```
+yarn regenerate
```

Comment:
Maybe a note about switch to `packages/apis` directory will be useful:

```suggestion
cd packages/apis

yarn regenerate
```

Reply:
it is relative to the markdown file

Reply:
Ok, make sense